config.toml

```diff
@@ -14,11 +14,12 @@ unit = "GB"
 alert = 50
 warning = 40
 
-[[block]]
-block = "memory"
-display_type = "memory"
-format_mem = "{mem_used_percents}"
-format_swap = "{swap_used_percents}"
+# Uncomment below to enable battery
+# [[block]]
+# block = "memory"
+# display_type = "memory"
+# format_mem = "{mem_used_percents}"
+# format_swap = "{swap_used_percents}"
 
 [[block]]
```

Comment:
This is the section that needs to be disabled I guess?

Reply:
:rofl: